test/components/Home.vue

```diff
@@ -89,7 +89,7 @@ export default {
       credential: {
         '@context': 'https://w3id.org/credentials/v1',
```

Comment:
Out of scope of this PR, but could update to a modern example with proper contexts.

Reply:
made an issue for it: https://github.com/digitalbazaar/bedrock-vue-credential-card/issues/25